### PR TITLE
Reader: migrate full post content styles to `reader-full-post` block

### PR DESCRIPTION
**Pull request by bluefuton**
The new Reader full post is nearly ready to release to all users, but it still relies on a bunch of content styles in the old full post (`reader/full-post/_style.css`).

This PR migrates the styles from the old stylesheet and stops including the old full post stylesheet in the SASS build.

⚠️ This should not be merged until the new full post has launched for all users.

### To test

Load a selection of posts in Reader full post view (e.g. https://wpcalypso.wordpress.com/read/feeds/19198732/posts/1191108767) and ensure there are no regressions with the styling of content.

### CODE REVIEW

**Comment by bluefuton** [comment]
@jancavan would you mind having a play, and as a bonus, drop any redundant style rules you spot?

I've brought across everything that targeted `reader__full-post-content` from the old stylesheet.

**Comment by bluefuton** [comment]
## todo
- Remove `.reader__full-post-content` class and retarget styles at `.reader-full-post__content`

File: client/reader/stream/post.jsx [hunk 1]
```diff
@@ -363,7 +363,7 @@ const Post = React.createClass( {
 				{ shouldUseFullExcerpt
 					? <EmbedContainer>
 							<div key="full-post-inline"
-								className="reader__full-post-content"
+								className="reader-full-post__story-content"
```
Comment (bluefuton) [inline client/reader/stream/post.jsx]: I know the stream shouldn't be using this class, but historically it has shared content styles with the full post view. We'll phase this out when we launch the refreshed stream cards later this year.

**Comment by jancavan** [comment]
This looks good @bluefuton. I think we eventually need to fix how the containers are being named. The current hierarchy is confusing:

-`reader-full-post__content`
--`reader-full-post__story` (article tag)
--- `reader-full-post__story-content reader__full-post-content`

`reader__full-post-content` should probably just be removed, but I'm concerned it might cause issues so we can tackle that in a separate PR.

**Comment by bluefuton** [comment]
> reader__full-post-content should probably just be removed

Should be gone now :) I removed it in https://github.com/Automattic/wp-calypso/pull/8892/commits/0309f4e65171daaf5826a3ae9ee440443f8c0e53#diff-e4e8dfe9d856c8280a2d8f4e602e654bR315.

**Comment by jancavan** [comment]
LGTM 👍

**Pull request merged**